src/pylorax/monitor.py

```diff
@@ -99,25 +134,13 @@ def iserror(self, line):
         """
         if "IGNORED" in line:
             return
-        simple_tests = ["Traceback (",
-                        "Out of memory:",
-                        "Call Trace:",
-                        "insufficient disk space:",
-                        "Not enough disk space to download the packages",
-                        "error populating transaction after",
-                        "traceback script(s) have been run",
-                        "crashed on signal",
-                        "packaging: Missed: NoSuchPackage",
-                        "packaging: Installation failed",
-                        "The following error occurred while installing.  This is a fatal error"]
-        re_tests =     [r"packaging: base repo .* not valid",
-                        r"packaging: .* requires .*"]
-        for t in simple_tests:
+
+        for t in LogRequestHandler.simple_tests:
```

Comment:
Please, use `self.simple_tests` here to use the simple tests of the current class.

Reply:
Fixed.